kex

```diff
@@ -43,7 +45,10 @@ EOF
 }
 
 po_list() {
-  kubectl get pods -o=jsonpath='{range .items[*].metadata.name}{@}{"\n"}{end}'
+  local NS_PARAM="--namespace=${NAMESPACE:-}"
```

Comment:
at first glance, I was under the impression that the [logic to process args](https://github.com/farmotive/kex/blob/ed0e3d115f924d258d380f90cb7e3be275b08824/kex#L74-L82) didn't enforce an order. It feels like the arg order should not matter.